graphql/schemabuilder/types.go

```diff
@@ -15,7 +15,7 @@ type Object struct {
 	Methods     Methods // Deprecated, use FieldFunc instead.
 	key         string
 	ServiceName string
-	IsFederated bool
+	IsRoot      bool
```

Comment:
Should we move this change to one of the previous commit?

Reply:
yes!